CZ1016_Intro to Data Science/2020/README.md

```diff
@@ -1,5 +1,5 @@
 # Introduction to Data Science
 
-| Group Members              | Link to Project Repo                                       |
-| -------------------------- | ---------------------------------------------------------- |
-| Bhargav, Kah Ee, Nicolette | [CatsnDogs](https://github.com/majulahsingapuri/CatsnDogs) |
+| Group Members              | Link to Project Repo                                                |
+| -------------------------- | ------------------------------------------------------------------- |
+| Bhargav, Kah Ee, Nicolette | [CatsnDogs](https://github.com/majulahsingapuri/CatsnDogs)          |
```

Comment:
I know in your commit history you added your project to the wrong file. Do try to refrain from such commits in the future that just change the layout but not add content.

Reply:
@johnsonfoo yes i do have a git hook set up but it doesnt work for markdown table formatting for some reason. im not sure if the hook works on your system when you commit code

Reply:
Started a new issue to track this request